### ISSUE TITLE
Multi Monitor Suppor

### ISSUE THREAD

**Issue by chrome1wave** [cwc/web-page-screensaver#34]
Can you please make an update so this will work on multiple monitors?  It needs to work in preview mode.  Thank you!

**Comment by theCheek** [comment]
yeah +1 on this. Currently using Nvidia Surround triple screen, and when the screensaver starts it immediately deactivates. This is when previewing and allowing it to run after configured timeout.
